test/doc/algo/transform.cpp

```diff
@@ -0,0 +1,34 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include <tuple>
+#include <eve/algo/concepts/relaxed.hpp>
+#include "detail/print.hpp"
+
+// an accurate linspace with transform and lerp
+template < eve::algo::relaxed_range R, eve::scalar_value T,  eve::scalar_value U> R linspace(R& r, T a, U b )
```

Comment:
This should go in `examples`, `using_existing`

Reply:
I think it's in doc/ so we can add them to the doxygen page like we did for all other functions.